openhands/core/main.py

```diff
@@ -35,8 +35,8 @@ class FakeUserResponseFunc(Protocol):
     def __call__(
         self,
         state: State,
-        encapsulate_solution: bool = ...,
```

Comment:
I'm not sure what these changes are for

Would love to have a second opinion

Reply:
These were to fix linter errors that were surfaced because `openhands.core.main()` is called by the resolver.